src/components/data_search/DatasetSearchTable.js

```diff
@@ -148,7 +148,7 @@ export const DatasetSearchTable = (props) => {
               truncate: true,
             },
             {
-              value: entry[0].study.dataCustodianEmail.join(', '),
+              value: entry[0].study?.dataCustodianEmail.join(', '),
```

Comment:
IIUC, this will break on an undefined study, not an undefined `dataCustodianEmail`. If we move the `?` to `study.dataCustodianEmail?.join`, we'll still get an `undefined` passed into the `join` method which will fail.

Reply:
Whoops I put it on the wrong side. According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining , it will short circuit and not run the join if the email is undefined.